pennylane/ops/op_math/condition.py

```diff
@@ -686,19 +686,16 @@ def _(*all_args, jaxpr_branches, n_consts_per_branch, n_args):
         args = all_args[n_branches : n_branches + n_args]
         consts_flat = all_args[n_branches + n_args :]
 
-        if isinstance(conditions[0], qml.measurements.MeasurementValue):
-            # if len(conditions) > 2:
-            #     raise ConditionalTransformError(
-            #         "'elif' branches are not supported when not using @qjit and the "
-            #         "conditional includes mid-circuit measurements."
-            #     )
-            # conditions = (conditions[0], ~conditions[0])
-            if any(not isinstance(pred, MeasurementValue) for pred in conditions[:-1]):
+        # Find predicates that use mid-circuit measurements. We don't check the last
+        # condition as that is always `True`.
+        mcm_conditions = [pred for pred in conditions[:-1] if isinstance(pred, MeasurementValue)]

```

Comment:
This happens inside the wrapper returned by `qml.cond` (see [here](https://github.com/PennyLaneAI/pennylane/blob/a72f3073d35ebe848e8c98c8ecb9fb317108c0c0/pennylane/ops/op_math/condition.py#L245)). It's just added for the false condition. Since the impl returns immediately once we find the first true condition, it was just easy to do this, since we iterate through all conditions/jaxprs zipped together.